mapserver.h

```diff
@@ -937,6 +939,13 @@ The :ref:`QUERYMAP <querymap>` object.
 Instances of querymapObj are always are always embedded inside the :class:`mapObj`.
 */
   typedef struct {
+#ifdef SWIG
+    %immutable;
+#endif /* SWIG */
+    struct mapObj *map; ///< Reference to parent :class:`mapObj`                                                                                                                 
```

Comment:
May not be required if only used to get the min and max size when loading the map (see comment above).